e2e/fixtures/rsc-asset/waku.config.ts

```diff
@@ -1,7 +1,6 @@
-/// <reference types="vite/client" />
+/// <reference types="waku/types" />
 
-import { defineConfig } from 'waku/config';
-import type { Plugin } from 'vite';
+import { defineConfig, VitePlugin } from 'waku/config';
```

Comment:
I thought tsc would complain unless we do:
```suggestion
import { defineConfig, type VitePlugin } from 'waku/config';
```

Reply:
Oh, I see. We have different tsconfig.json in fixtures (and examples).